src/Domain/Core/Stager/Stager.php

```diff
@@ -46,7 +46,7 @@ private function validateCommand(array $composerCommand): void
         }
 
         if (reset($composerCommand) === 'composer') {
-            throw new InvalidArgumentException('The Composer command cannot begin with "composer"--it is implied');
+            throw new InvalidArgumentException('The Composer command should not begin with "composer"--it is implied');
```

Comment:
It seems to me that if it's throwing an exception, _cannot_ is the correct language. "Should not" implies that it's still possible.